src/pages/team-building/ui/MyTeam.tsx

```
@@ -0,0 +1,297 @@
+            })}
+          </Swiper>
+
+          {cardData.state === 'hasData' &&
+            filteredMembers?.map(member => (
+              <div
+                className={cn(
+                  'flex justify-between items-center p-[14px_20px_24px] bg-white border-b border-[#EEEEEE]',
+                )}
+              >
+                <div className="flex gap-2">
+                  <div className="w-24 h-24 rounded-[10px] bg-[#E9E9E9] overflow-hidden relative">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)